xbmc/xbmcgui.go

```diff
@@ -155,7 +155,7 @@ func Notify(header string, message string, image string) {
 // InfoLabels ...
 func InfoLabels(labels ...string) map[string]string {
 	var retVal map[string]string
-	executeJSONRPC("GetInfoLabels", &retVal, Args{labels})
+	executeJSONRPC("XBMC.GetInfoLabels", &retVal, Args{labels})
```

Comment:
`GetInfoLabels` without `XBMC.` is not working through RPC?

Reply:
no, https://github.com/elgatito/elementum/blob/c386f69f7611e1df37787d53fe408181db8ccc65/osdb/util.go#L109 was always `nil` so then search was done via filename instead of labels.

Reply:
`DEBU  osdb         ▶ GetPayloads      Fetched VideoPlayer labels: map[string]string(nil)`